website/src/components/Messages/AdminMessageTable.tsx

```diff
@@ -106,5 +119,17 @@ const DateDiff = ({ children }: { children: string | Date | number }) => {
 };
 
 export const AdminMessageTable = (props: StrictOmit<DataTableProps<Message>, "columns">) => {
+  const { trigger: deleteMessage } = useSWRMutation("/api/admin/user_messages", (path, args: { arg: string }) =>
+    del(`/api/admin/delete_message/${args.arg}`)
+  );
+
+  async function handleDeleteMessage(id: string) {
+    if (confirm("Are you sure you want to delete this message?")) {
+      deleteMessage(id);
+    }
+  }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
You should not ignore eslint here. Please add the deps to 2nd arg

Reply:
Done - I change it to use a proper dialog too.